src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -46,11 +51,13 @@ public void Dispose()
 
         public Task StartAsync(CancellationToken cancellationToken)
         {
-            _poll = _folderSource.CreateNewFileWatcher(OnNewFile, _pollIntervalInSeconds > 0 ? _pollIntervalInSeconds : DefaultPollIntervalInSeconds);
+            _poll = _folderSource.CreateFileWatcher(_fileWatcherType, OnFileWatcher, nextItem: null, pollIntervalInSeconds: _pollIntervalInSeconds > 0 ? _pollIntervalInSeconds : DefaultPollIntervalInSeconds);
+
+            // TODO: need to decide what to do when _poll is null i.e. trigger folder does not exist.
```

Comment:
See how I handle this in FileTrigger https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions/Extensions/Files/Listener/FileListener.cs#L67 - I throw an error in this case. I think that's what you should do as well, rather than silently ignoring the issue.